builtin/commit-graph.c

```diff
@@ -146,8 +146,9 @@ static int graph_write(int argc, const char **argv)
 		flags |= COMMIT_GRAPH_WRITE_SPLIT;
 	if (opts.progress)
 		flags |= COMMIT_GRAPH_WRITE_PROGRESS;
-	if (opts.enable_changed_paths)
-		flags |= COMMIT_GRAPH_WRITE_BLOOM_FILTERS;
+	if (opts.enable_changed_paths || 
+		git_env_bool(GIT_TEST_COMMIT_GRAPH_CHANGED_PATHS, 0))
```

Comment:
nit: this should be `<tab><sp><sp><sp><sp>git_env_bool(`... to match the alignment of the line above after `if (` and then the `flags |=` line should be one tab deeper than the left of the `if`.